[WIP] Infiltration mode

astralloop opened the pull request:
:cl: Astral
add: Infiltration mode has been added!
add: Syndicate infiltrators have been assigned to your station, and will try to do their dirty work for the syndicate, from becoming heads of staff, to stealing power, to hijacking the AI!
/:cl:

astralloop commented:
I need someone to map a shuttle, I tried, it sucked.

kevinz000 commented:
i thought we decided stealthops weren't fun

cacogen commented:
Isn't this just traitor except you start on a ship and aren't on the crew manifest?

MrDoomBringer commented:
@kevinz000 <--someone who gets it

Shapsy commented:
60TC plus free pistol and cham gear what the

factoryman942 commented:
Murderbone traitors but they get more tc and can malf the ai

astralloop commented:
@factoryman942 they're not meant to murderbone. in fact, i'm gonna add a message telling them not to.

optimumtact commented:
This is a clone of nukeops that will play out exactly like nukeops with our playerbase, this should not be merged

MimicFaux commented:
The argument that this is a clone of nuke ops feels weak to me. We have traitors, but we also have internal affairs. This is a similar but comes with a different flavour, and I think has some great potential.

astralloop commented:
thanks @WJohn for the mapping

WJohn commented:
![](https://i.imgur.com/KKXM7LU.png)

Image of the base for those curious.

factoryman942 commented:
just had a round of it on bagil
i was comms agent watching over them
They had both "Hijack xxx ai unit" and "Steal a functional ai unit", but they reported not having an SEU / not knowing what it was or how to use it.
They also enquired whether they had to take the powersink with them after extracting the power.

WJohn commented:
Could you make it so the base doesn't have forced fullbright lighting? I gave it a bunch of bulbs for a reason.

kevinz000 commented on code/modules/antagonists/infiltrator/infiltrator.dm:
```diff
@@ -0,0 +1,89 @@
+	to_chat(owner, "<span class='notice bold'>Your job is to infiltrate [station_name()], and complete our objectives.</span>")
+	to_chat(owner, "<span class='notice'>You have an uplink implant, precharged with 30 TC. Use it wisely.</span>")
+	to_chat(owner, "<span class='notice'>You also have an internal radio, for communicating with your team-mates at all times.</span>")
+	to_chat(owner, "<font size=2><span class='notice bold'>Do NOT kill or destroy needlessly, as this defeats the purpose of an 'infiltration'!</span></font>")
```
don't forget to make a jobban type for this if you want this to be enforced at all.

astralloop replied:
added to jobban panel.

kevinz000 commented on code/modules/antagonists/hijacked_ai/hijacked_ai.dm:
don't forget to remove radio channel.

astralloop replied:
done.

astralloop commented:
I have added a new Major objective, Kidnap. the target will either be a head of staff or security staff. I have also added a small jail cell to the base.

astralloop commented:
![jailcell](https://user-images.githubusercontent.com/36741211/37205658-8e511dba-2363-11e8-925a-27abd0ef135a.PNG)

WJohn commented:
👎 on the kidnap objective. It is a total pain in the ass to kidnap someone in the first place, involves a whole hell of a lot of being restrained and unable to do anything, and can be completely negated by committing suicide at any time. It's a bad objective, gameplay wise.

astralloop commented:
@WJohn actually, i made the objective still count if they suicide

AlexTheSysop42 commented:
[Engineering] Danny Barnes asks, "I'm getting an atmos alarms from ''Syndicate Infiltration Crusier'', is this normal?"

also the shuttle cant leave once it goes back to the base

Supermichael777 commented:
you managed to make team-ling twice as cancerous by removing the slim odds one might be another's target to a full stealth team that has total trust and every tool they need. Good job.

BeeSting12 commented:
The AI hijacker shouldn't be able to be applied multiple times, make it only able to be done once and trying again says it's already in progress to prevent deadchat spam.

astralloop commented:
@BeeSting12 there was a bug that allowed the hijacker to stay in your hands after applying it

Robustin commented:
I think you should give this a more definitive conclusion once the Syndies greentext.

Switching to extended when the Antags WIN hardly seems like a satisfying conclusion for ANYONE.

PolyGanon commented:
Minor issue with the ship itself- The airlocks don't seem to be cycle linked (at least in the version test-merged on Bagil), causing spacewind to regularly empty at least one side of the ship. Would Tiny Fans be a worthwhile addition?

astralloop commented:
@PolyGanon tiny fans added

Cyberboss commented:
@MapDiffBot

coiax commented:
Tiny fans are evil reeee

astralloop commented:
@coiax They... are... not!
